Create README.md Pull Request

Avto555 opened the pull request:
test of Pull request

Avto555 committed:
homowork4

not finished yet

Avto555 committed:
homework4

Avto555 committed:
changes

Michael-Malania commented on Homeworks/Homework2/Homework2.js:
```diff
@@ -0,0 +1,12 @@
+//let array1 ['yellow',23,77,123,'blue'] ??
+
```
ეს ფაილი თუ არაა საჭირო წაშალე საერთოდ

Michael-Malania commented on Homeworks/Homework3/Task1.js:
```diff
@@ -0,0 +1,12 @@
+let numbers =[1,2,3,4,5,6,7,8,9,10]
+
+let NonEvenNunmbers = numbers.filter(i => {
+    if (i % 2 === 1) {
```
აქ ჯობია დავწეროთ არ უდრის 0-ს და არა უდრის თუ არა ერთს

Michael-Malania commented on Homeworks/Homework3/Task2.js:
```diff
@@ -0,0 +1,6 @@
+let unsortedstrings =["kmaioqwemkqwe"]
+
+
+console.log(unsortedstrings.sort);
```
არ არის სწორი, ზოგადად ფუნქციას ფრჩხილები უნდა გააჩნდეს აქ ამას ვერ ვხედავ, და ასევე სორტირებას ვუკეთებთ სტრიქონს და არა მასივს ამიტომაც ამ სტრიქონის მასივში ჩასმა არ მოგვცემს არაფერს. ძველ ჩანაწერებს გადახედე და ესეც წაიკითხე https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort

Michael-Malania commented on Homeworks/Homework4/task1.js:
```diff
@@ -0,0 +1,10 @@
+
+const array = [1, 2, 3, 4, 10, 11];
+
+for (var i of array) {
```
აქ var i არ გამოიყენო გამოიყენე let i

Michael-Malania commented on Homeworks/Homework4/task1.js:
```diff
@@ -0,0 +1,10 @@
+const array = [1, 2, 3, 4, 10, 11];
+
+for (var i of array) {
+  if (i == 10) {
```
ეს ჯობია ზევით დავწეროთ, მაგ. let userRequest = 10, და შემდეგ if(i === userRequest), ხო ასევე ორი ტოლია გიწერია არადა სამი ტოლია უნდოდა, რადგან სამი ტოლია ამოწმებს ცვლადის მნიშვნელობებს რაც შენ გჭირდება ხოლო ორი ტოლია სხვა რამეა, ეგ ყოველთვის ცდილობს ჯერ შესადარებელი ორი მონაცემის ტიპი გადააკონვერტიროს ერთი მონაცემის ტიპზე და შემდეგ მოახდინოს შედარება მაგ "5"==5 მოგცემს ჭესმარიტ მნიშვნელობას.

Michael-Malania commented on Homeworks/Homework4/task6.js:
```diff
@@ -0,0 +1,8 @@
+  return str.trim().length === 0;
+}
+
+console.log("არის სტრინგი ცარიელი?: " + isStringEmpty("gfhfghf"));
```
ამ პროგრამას გამოაქვს true თუ არის ცარიელი და false თუ არ არის ცარიელი რაც კარგია, მაგრამ მე მინდა გამოიტანო ტექსტი, თუ ცარიელია რომ უთხრას მომხმარებელს ცარიელიათქო და თუ არ არის ცარიელი რომ უთხრას არაა ცარიელიო

Michael-Malania commented on Homeworks/Homework4/task8.js:
```diff
@@ -0,0 +1,16 @@
+
+console.log(hideEmail(x));
+
+// რამდენი წერტილიცაა იმდენს ვერ ვაკეთებ.
```
მართალი ხარ, ამ ლოგიკით თუ რაიმე განსხვავებული სიმბოლო არ იქნება ემაილში მაშინ ვერ გააკეთებ, მაგრამ მე მეწერა giorgi.beridze ანუ განმასხვავებლად სახელისა და გვარის იყო წერტილი, შესაბამისად თუ გექნება ესეთი ემაილი სადაც წერტილი ან რაიმე სხვაა გამოყენებული დაუშვათ "-", "_", მაშინ შეგიძლია გაიგო მაგ სიმბოლოდან @-მდე რამდენია სტრიქონის ზომა და დანარჩენი მოდი შენ მოიფიქრე და გადაასწორე კოდი

Michael-Malania reviewed (changes requested):
ავთო ჩემი კომენტარები ნახე შესაბამისად გაასწორე და შემდეგში დავმერჯავ თუ მომეწონება, და ასევე PR-ბს ხსნი არასწორად ერთ PR-ში ფუშავ ყველა ჩეინჯებს არადა მე გითხარი რომ ცალ ცალკე პიარში დაგეფუშა თითოეული დავალების ჩეინჯები ეს გამოსაწორე სამომავლოდ აუცილებლად.

Avto555 committed:
Add files via upload

Javascript-project for Academy of Digital Industries

პროექტი ატვირთულია და ფუნქციონირებს netlify.com-ზე: https://avtojsproject.netlify.app/

მთავარ გვერდზე არის გაკეთებული რამოდენიმე ანიმაცია, ტექსტების hover-ზე ხდება მათი "გადალაგების" ეფექტი რომელიც გაკეთებულია ჯავასკრიპტით. არის hamburger მენიუ. კარუსელზე არის დასმული ცნობილი გამონათქვამენის API. ღილაკების დაჭერაზე გვაჩვენებს მომდევნო გამონათქვამს თავისი ავტორით. (ანუ დავალებაში მოთხოვნილი კარუსელი და API მაქვს ერთად გაკეთბული)

რამოდენიმე გვერდზეა გამოყენებული Bootstrap 5.3

გაკეთებულია ავტორიზაციის და რეგისტრაციის გვერდები, მონაცებემი ინახება LocalStorage-ში. მომხმარებელმა აუცილებლად უნდა გამოიყენოს მინიმუმ 8 ასო და 1 სიმბოლო მაინც იმისათვის რომ დარეგისტრირდეს.

footer-ში არ არის საკონტაქტო ინფო თუმცა თუ იუზერი გადავა "ჩემს შესახებ" გვერდზე ექნება წვდომა სრულ ინფორმაციაზე ასევე შეძლებს CV-ის გადმოწერას

გამოყენებული წყაროები: ფერების პალეტი - coolors.co ფონტი - fonts.google.com და fonts.ge ტექსტის "გადალაგება" youtube.com 2 ღილაკის CSS - https://copy-paste-css.com/

დანარჩენი არის ლექციებზე და გარე წყაროებიდან შეძენილი ცოდნის გამოყენება

Avto555 committed:
Create readme.md

Avto555 committed:
Update readme.md
